runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -206,6 +210,12 @@ func s:StartDebug_internal(dict)
     call win_gotoid(curwinid)
   endif
 
+  if s:GetVariablesWindow()
+    let curwinid = win_getid(winnr())
```

Comment:
To get the id of the current window, win_getid() can be used without any arguments.  No need to call winnr() here.